tasks/postgres.yml

```diff
@@ -0,0 +1,24 @@
+
+- name: Add peertube user in Postgres
+  become: yes
+  become_user: postgres
+  postgresql_user:
+    name: peertube
```

Comment:
Use defaults variables

Reply:
[Here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/defaults/main.yml#L17) add peertube_dbuser.
[Here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/postgres.yml#L6) and [here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/postgres.yml#L14) it is used.